test/integration_test.py

```diff
@@ -713,3 +718,52 @@ def test_mini_dss_failures(self):
                 mini_dss.get_file(uuid, version, 'aws')
             with self.assertRaises(self.SpecialError):
                 mini_dss.get_native_file_url(uuid, version, 'aws')
+
+
+class AzulChaliceLocalIntegrationTest(AzulTestCase):
+    host = '127.0.0.1'
+    port = 8000
+    url = furl(scheme='http', host=host, port=port)
+    server = None
+    server_thread = None
+
+    @classmethod
+    def setUpClass(cls, config=config) -> None:
+        super().setUpClass()
+        app_module = load_app_module('service')
+        app_dir = os.path.dirname(app_module.__file__)
+        factory = chalice.cli.factory.CLIFactory(app_dir)
+        config = factory.create_config_obj()
+        cls.server = factory.create_local_server(app_obj=app_module.app,
+                                                 config=config,
+                                                 host=cls.host,
+                                                 port=cls.port)
```

Comment:
```suggestion
                                                 host=cls.url.host,
                                                 port=cls.url.port)
```

and eliminate the two attributes.